REPOSITORY-GUIDELINES.md

```diff
@@ -1,101 +1,189 @@
+  can occur post-transfer, but should ideally occur shortly thereafter.
+- An `AUTHORS` file should be created in the repo root, if one does not already
+  exist, that includes the original authors of the code. See
+  [knative/serving AUTHORS](https://github.com/knative/serving/blob/master/AUTHORS)
+  for reference.
+  - Note that copyright notices should only be modified or removed by the people
+    or organizations named in the notice.
+
+## Removing Repositories
+
+As important as it is to add new repositories, it is equally important to prune
+repositories when necessary. See [Grounds for removal](#grounds-for-removal).
+
+It is important to the success of Knative that all Knative repositories stay
+active, healthy, and aligned with the scope and mission of project.
+
+### Grounds for removal
+
+Core repositories may be removed from the project if they are deemed _inactive_.
+Inactive repositories are those that meet any of the following criteria:
+
+- There are no longer any active maintainers for the project, and no
+  replacements can be found.
+- All PRs and issues have gone un-addressed for longer than six months.
+- There have been no new commits or other changes in more than a year.
+- The contents have been folded into another actively maintained project.
+
+### Procedure for removal
+
+When a repository has been deemed eligible for removal, we take the following
+steps:
+
+- A proposal to remove the repository is brought to the attention of the
+  Technical Oversight Committee through a GitHub issue posted in the
+  [docs](https://github.com/knative/docs) repo.
+  - Feedback is encouraged during a Technical Oversight Committee meeting before
+    any action is taken.
+- Once the TOC has approved of the removal, if the repo is not moving to another
+  actively maintained project:
+  - The repo description is edited to start with the phrase "[EOL]"
+  - All open issues and PRs are closed
+  - All external collaborates are removed
+  - All webhooks, apps, integrations, or services are removed
+  - GitHub pages are disabled
+  - The repo is marked as archived using GitHub's archive feature
+  - The removal is announced on the knative-dev mailing list
+
+This procedure maintains the complete record of issues, PRs, and other
+contributions. It leaves the repository read-only and makes it clear that the
+repository is retired and no longer maintained.
+
+## Additional / Experimental Repositories
+
+We want Knative working groups to be able to own code outside the core `knative`
+organization, which is kept intentionally small and has a high bar for entry.
+The `knative-sandbox` github organization was created for this purpose.
+Repositories in `knative-sandbox` are intended to give working groups more
+latitude to experiment with new ideas and to self-organize and manage
+contributions which may be important to the project but which do not need the
+level of governance of the core project.
```

Comment:
how do we distinguish between experimentation vs production-ready repositories?

Reply:
I'm assuming that it would start with a README, but that api group versions might be one hint.

Reply:
I would expect that all repos follow the same conventions and wording in the README. Maybe recommending which badges to use might be good to.

Reply:
Do we need to get that text into this PR? If so, can you recommend some text?

If it's not blocking, I'd prefer to handle this with a followup PR

Reply:
it's not blocking. I'm thinking that the release badge is enough.